modules/yaml/yaml.go

```diff
@@ -35,13 +35,22 @@ func renderHorizontalHtmlTable(m yaml.MapSlice) string {
 		key := mi.Key
 		value := mi.Value
 
-		if key != nil && reflect.TypeOf(key).String() == "yaml.MapSlice" {
+		switch reflect.TypeOf(key).String() {
```

Comment:
This wasn't even something I wrote this time, so you could me asking me to change this all over Gogs if you wanted, but ok, will look into it.